rubygems-org-api.md

```diff
@@ -6,53 +6,54 @@ previous: /command-reference
 next: /run-your-own-gem-server
 ---
 
-<em class="t-gray">Details on interacting with RubyGems.org over HTTP.</em>
-
-> NOTE: The API is a work in progress, and [can use your help!](https://github.com/rubygems/gemcutter)
-> RubyGems itself and the
-> [gemcutter gem](https://rubygems.org/gems/gemcutter) use the API to push gems,
-> add owners, and more.
-
-* [API Authorization](#api-authorization): How to authenticate with RubyGems.org
-* [Gem Methods](#gem-methods): Query or create gems to be hosted
-* [Gem Version Methods](#gem-version-methods): Query for information about
-  versions of a particular gem
-* [Gem Download Methods](#gem-download-methods): Query for download statistics
-* [Owner Methods](#owner-methods): Manage owners for gems
-* [Webhook Methods](#webhook-methods): Manage notifications for when gems are
-  pushed
-* [Activity Methods](#activity-methods): Query for information about site-wide
-  activity
-* [Misc Methods](#misc-methods): Various other interactions with the site
-
-API Authorization
------------------
-
-Some API calls require an Authorization header. To find your API key, click on
-your username when logged in to [RubyGems.org](http://rubygems.org) and then click on 'Edit Profile'. Here's an example of
-using an API key:
+<em class="t-gray">HTTP를 통한 RubyGems.org과의 통신 자세히 보기</em>
+
+> 주의: API는 작업중이고, [도움이 필요합니다!](https://github.com/rubygems/gemcutter)
+> RubyGems 자체와
+> [gemcutter gem](https://rubygems.org/gems/gemcutter)은 gem을 넣고 소유자를
+> 추가하고 그 밖의 다른 일을하기 위해 API를 사용합니다.
+
+* [API 인증](#api-authorization): RubyGems.org에 인증하는 방법
+* [Gem 메소드](#gem-methods): 호스트할 gem을 만들거나 조회하기
+* [Gem 버전 메소드](#gem-version-methods): 특정 gem의 버전에 관한 정보를
+  조회하기
+* [Gem 다운로드 메소드](#gem-download-methods): 다운로드 통계 조회하기
+* [소유자 메소드](#owner-methods): gem의 소유자를 관리하기
+* [웹훅 메소드](#webhook-methods): gem이 넣어졌을 때의 알림 관리하기
```

Comment:
영향 범위는 이렇네요.
```
contributing.md:171:pushed. Currently powers [m.rubygems.org](http://m.rubygems.org) and
make-your-own-gem.md:127:Once this has been setup, you can push out the gem:
make-your-own-gem.md:129:    % gem push hola-0.0.0.gem
make-your-own-gem.md:310:Finally, to get Hola's executable included when you push the gem, you'll need
make-your-own-gem.md:323:> Note that you should change the gem's version when pushing up a new release.
make-your-own-gem.md:441:when you push a gem, [RubyDoc.info](http://rubydoc.info/) generates YARDocs
patterns.md:68:Suppose you have a 'stack' gem that holds a `Stack` class with both `push` and
patterns.md:78:* **Version 1.1.0**: `push` now returns the value pushed (it used to return
plugins.md:31:* [PushSafety](#pushsafety)
plugins.md:200:<a id="pushsafety"> </a>
plugins.md:204:[https://github.com/jdleesmiller/push_safety](https://github.com/jdleesmiller/push_safety)
plugins.md:206:Applies a whitelist to `gem push` to prevent accidentally pushing private gems to the public RubyGems repository.
publishing.md:17:* [Push Permissions on RubyGems.org](#push-permissions-on-rubygemsorg)
publishing.md:55:RubyGems 2.2.0 and newer support the `allowed_push_host` metadata value to
publishing.md:56:restrict gem pushes to a single host.  If you are publishing private gems you
publishing.md:57:should set this value to prevent accidental pushes to rubygems.org:
publishing.md:61:      s.metadata['allowed_push_host'] = 'https://gems.my-company.example'
publishing.md:79:After creating the account, use your email and password when pushing the gem.
publishing.md:85:    $ gem push squid-utils-0.1.0.gem
publishing.md:101:maintainers permission to push the gem to rubygems.org through the [gem
```